.github/workflows/generateAutoConfigs.yaml

```diff
@@ -0,0 +1,83 @@
+name: Generate Spring Auto-Configurations
+
+on:
+  push:
+    branches:
+      - 'main'
+  # what triggers this action is yet TBD - leaving PRs and `main` pushes for now
```

Comment:
We can maybe switch to [`on: workflow_dispatch`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch) for now to just [manually trigger](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow?tool=webui) this workflow.

Reply:
Thanks @emmileaf - I modified the trigger to _only_ use `workflow_dispatch`